### PR TITLE
Added clarification of using change tracking policy on entities with embeddables

### PR DESCRIPTION
**Pull request by carnage**
Spend a good long time figuring this one out today; figured it'd help someone else in the future

### CODE REVIEW

File: docs/en/reference/change-tracking-policies.rst [hunk 1]
```diff
@@ -129,6 +129,32 @@ The check whether the new value is different from the old one is
 not mandatory but recommended. That way you also have full control
 over when you consider a property changed.
 
+If your entity contains an embeddable, you will need to notify 
+separatly for each property in the embeddable when it changes
```
Comment (greg0ire) [inline docs/en/reference/change-tracking-policies.rst]:
```suggestion
separately for each property in the embeddable when it changes 
```

File: docs/en/reference/change-tracking-policies.rst [hunk 2]
```diff
@@ -129,6 +129,32 @@ The check whether the new value is different from the old one is
...
+
+    <?php
+    // ...
+    
+    class MyEntity implements NotifyPropertyChanged
+    {
+        public function setEmbeddable(MyValueObject $embeddable)
+        {
+            if (!$embeddable->equals($this->embeddable)) {
+                //notice the entityField.embeddableField notation for referencing the property
```
Comment (greg0ire) [inline docs/en/reference/change-tracking-policies.rst]:
```suggestion
                // notice the entityField.embeddableField notation for referencing the property
```

File: docs/en/reference/change-tracking-policies.rst [hunk 3]
```diff
@@ -129,6 +129,32 @@ The check whether the new value is different from the old one is
...
+                $this->_onPropertyChanged('embeddable.prop1', $this->embeddable->getProp1(), $embeddable->getProp1());
+                $this->_onPropertyChanged('embeddable.prop2', $this->embeddable->getProp2(), $embeddable->getProp2());
+                $this->embedable = $embeddable;
```
Comment (greg0ire) [inline docs/en/reference/change-tracking-policies.rst]:
```suggestion
                $this->embeddable = $embeddable;
```

**Review by greg0ire — CHANGES_REQUESTED** [review]

**Comment by carnage** [comment]
@greg0ire Noticed this hadn't been merged; rebased & retargeted to 2.9 should be good to go in.

**Review by greg0ire — APPROVED** [review]

**Review by SenseException — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you @carnage!

**Pull request merged**